src/celeritas/g4/EmExtraPhysicsHelper.hh

```diff
@@ -0,0 +1,41 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/g4/EmExtraPhysicsHelper.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+
+class G4GammaNuclearXS;
+
+namespace celeritas
+{
+
+//---------------------------------------------------------------------------//
+/*!
+ * A helper class to interface with Geant4 cross sections.
```

Comment:
I don't understand the purpose or interface of this class. Are you expecting to add more methods to it? Is it supposed to have the same interface as a particular Geant4 class? Will it be used outside of the GammaNuclearModel data construction?

Reply:
Yes, I plan to add additional methods for other cross sections used in the ElectroNuclear and potentially the MuonNuclear models. Nevertheless, I am open to adopting an alternative approach if there is a better way to provide a similar wrapper to corresponding Geant4 methods.

Reply:
I added a short class description and will expand it as I add more methods.